python-checks/src/test/resources/checks/redundantJump.py

```diff
@@ -250,7 +251,51 @@ def redundant_return_inside_else_block():
         print(e)
     else:
         print(42)
-        return # Noncompliant
+        return # FN
+
+def non_redundant_return_inside_except():
+    try:
+        pass
+    except E as e:
+        print(e)
+        return # OK
+    print("after try")
+
+def non_redundant_return_inside_else():
+    try:
+        pass
+    except E as e:
+        print(e)
+    else:
+        return # OK
```

Comment:
We should add a statement before the `return`, otherwise the exception on single statements in a block applies here.